messaging/src/main/java/io/lantern/messaging/Messaging.kt

```diff
@@ -343,13 +346,21 @@ class Messaging(
         }
         replyToSenderId?.let { msgBuilder.setReplyToSenderId(it) }
         replyToId?.let { msgBuilder.setReplyToId(it) }
-        val out =
-            Model.OutboundMessage.newBuilder().setMessageId(base32Id)
+        var out: Model.OutboundMessage.Builder? = null
+        if(recipientId != myId.id){
```

Comment:
Please run `ktlintFormat`

Reply:
hhhmm... my bad... I forget to run the auto formatter after any change 😰